### ISSUE TITLE
[ERROR] Error connecting to streamdeck: Hid(HidApiError { message: "No HID devices with requested VI D/PID found in the system." })

### ISSUE THREAD

**Issue by hellishvictor** [ryankurte/rust-streamdeck#28]
Hi, I get the error on the title using one of the followings:

` streamdeck-cli --serial CL1234567890 get-buttons` 
` streamdeck-cli --vid 0fd9 get-buttons` 

However, it did connect using the "--pid" option:
streamdeck-cli --pid 0063 get-buttons
` Connected to device (vid: 0fd9 pid: 0063 serial: CL1234567890)`

Cheers.

**Comment by hellishvictor** [comment]
Same with the SD XL (pid 006c) using the serial and vid, but no problem using the pid.

**Comment by AleksanderZdunek** [comment]
Yes, with the XL this is expected behaviour. Pid is required parameter.

Technically vid is required as well, but currently there is no reason to use anything but the default value.
Serial is optional and only really useful to distinguish between StreamDecks if you have more than one of the same model connected at the same time. But even when using serial you still need to pass the correct pid as well.

**Comment by hellishvictor** [comment]
Ok, --serial and --vid are complementary of --pid (which is the main) for be used only when there's several devices connected.
Thanks for your explanation.
